snhuproduct/turbine
# SNHUT-2514/ViewLearnerProfileInformation

### Description
Learner profile education tab. 
To test: 
Navigate to this url: http://localhost:4200/learner-profile/c75UZCzlGk85q6TNR864
Click on the Education tab. You can edit and view the learner's profile info. 

Some renaming of files were done to conform to standard. 

The APi for employment status needs to be worked on by the backend for uniformity. On the api, we have : unemployed, full-time, part-time, seeking work . but the submit values require 'seeking-employment'. 
My thoughts. The 'seeking work' should be changed to 'seeking-employment'


- Please write a short description of your changes here. Provide as much context as needed.

### Video or Screenshots

#### Video

- Paste your link here.

#### Screenshots

| Original | Updated |
| -------- | ------- |
|          |         |

### Other Details

- [JIRA Ticket](https://JIRA_LINK_HERE)

#### Checklist

- [ ] **Assumptions of User Story met (including acceptance criteria)**
- [ ] **All new necessary Unit tests were CREATED** to promote code coverage
- [ ] **All required unit tests are PASSING**
- [ ] The proposed functionality was tested manually locally, and it's working fine.
- [ ] PR was properly reviewed by a team member (at least 1 approval required to merge!)
- [ ] Unnecessary console.logs AND/OR comments were removed
- [ ] Feature ok-ed by UX designer (If needed - not for back end components) and Product Owner
- [ ] Ensure configuration and/or build changes documented
